lib/core/util.js

```diff
@@ -816,6 +817,12 @@ function errorRequest (client, request, err) {
   }
 }
 
+function maybeMarkAsUncloneable (target) {
+  if (markAsUncloneable !== undefined) {
+    markAsUncloneable(target)
+  }
+}
```

Comment:
This should be added to `lib/web/fetch/webidl.js`

Reply:
Should it stay as a separate function? Feels weird if I attach it to `webidl.util` as it's not part of the WebIDL spec.

Reply:
I think it is, at least in part, because the webidl declaration defines if an object can be serialized. It also defines platform objects. webidl.util.markAsUncloneable or webidl.markAsUncloneable are both fine.